### DISCUSSION TITLE
如果同一个jvm 进程中，支持同时连接两个不同的kerberos hadoop 集群？

### DISCUSSION THREAD

**Discussion by melin** [apache/kyuubi#5495] [Q&A]
如果同一个jvm 进程中，支持同时连接两个不同的kerberos hadoop 集群？

**Comment by yaooqinn** [comment]
This question is regarding Hadoop. It's unclear if Kyuubi members are knowledgeable about this topic. There are user mailing lists available for Hadoop users.
